src/components/Table/GridTable.tsx

```diff
@@ -345,21 +302,20 @@ export function GridTable<R extends Kinded, S = {}, X extends Only<GridTableXss,
     function visit(row: GridDataRow<R>): void {
       const matches =
         filters.length === 0 ||
+        row.pin ||
         filters.every((filter) =>
           columns.map((c) => applyRowFn(c, row)).some((maybeContent) => matchesFilter(maybeContent, filter)),
         );
       // Even if we don't pass the filter, one of our children might, so we continue on after this check
       let isCard = false;
       if (matches) {
         isCard = nestedCards && nestedCards.maybeOpenCard(row);
-        if (isCard && nestedCards) {
-          maxCardPadding = nestedCards.maxCardPadding(maxCardPadding);
-        }
         filteredRows.push([row, makeRowComponent(row)]);
       }
 
       const isCollapsed = collapsedIds.includes(row.id);
-      if (!isCollapsed && row.children) {
+      if (!isCollapsed && !!row.children?.length) {
+        nestedCards && matches && nestedCards.addSpacer();
```

Comment:
This was important to have a spacer between the parent and child.

Reply:
Just checking but this is adding a spacer between the parent and the 1st child after it, but not any space after a child? I.e. it's

```
* parent 1
* spacer
* child 1
* child 2
* child 3
* parent 2
* ```

And not:

```
* parent 1
* spacer
* child 1
* child 2
* child 3
* spacer
* parent 2
* ```

?

I think that makes sense, but that we don't need space after a child, going into the next parent, but just checking.

Reply:
Yes, correct! The "after child" situation seemed to look good in the POC so I opted to not muck with it.